### PR TITLE
pass client options

### LINKED ISSUE #506: Pass client config to `new Hoodie(options)`
This allows, for example, to define `hoodie.client.url` in the `package.json`. For example, if your Hoodie app runs on https://myapp.com, you can load `<script src="https://myapp.com/hoodie/client.js"></script>` from any domain you want, it will connect to your Hoodie app automagically

### PR DESCRIPTION
**Pull request by gr2m**
closes #506

### CODE REVIEW

**Comment by gr2m** [comment]
this is ready for review @hoodiehq/maintainers

**Comment by HipsterBrown** [comment]
LGTM :+1:

File: server/bundle-client.js [hunk 1]
```diff
@@ -20,7 +20,8 @@ function bundleClient (config, callback) {
   stream.on('error', callback)
   stream.on('end', function () {
     // TODO: pass client configuration to constructor
```
Comment (courajs) [inline server/bundle-client.js]: You can remove this TODO now :)

**Comment by courajs** [comment]
LGTM

**Comment by gr2m** [comment]
thanks you two!

**Pull request merged**